config/presets/storage.json

```diff
@@ -1,11 +1,34 @@
 {
   "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^1.0.0/components/context.jsonld",
   "@graph": [
+    {
+      "@id": "urn:solid-server:default:MetadataController",
+      "@type": "MetadataController"
```

Comment:
It might make sense to move this one to a different preset file, so people can more easily plug in a different kind of metadata controller.

Reply:
Currently the MetadataController only contains a few utility functions, so it might make even more sense to just remove and and have a MetadataUtil file.